### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by thierer**
Some (imho) issues, I found during an (incomplete and cursorily) check today.

Also, there's are duplicate entries for "Basic Bär" (page 65), but I'm not sure which one you'd like to keep.

### CODE REVIEW

File: issues/8407/40 Pascal – leistungsfähiger und eleganter als Basic (Teil 1).html [hunk 1]
```diff
@@ -74,7 +76,7 @@ <h1>Pascal – leistungsfähiger und eleganter als Basic (Teil 1)</h1>
             <figcaption>Listing 2. Programm mit REPEAT-Schleife</figcaption>
         </figure>
 
-        <p>Die unterstrichenen Worte sind reservierte »Steuerworte« in Pascal, die nicht anders gebraucht werden dürfen. Die übrigen Worte sind Standardprozeduren. Man erkennt sie auch daran, daß sie ein oder mehrere Argumente in Klammern annehmen können. Ein Statement kann aus Steuerworten und Prozeduren bestehen, was man leicht erkennen kann, weil nach &lsquo;WHILE-&rsquo;..&lsquo;DO&rsquo; bekanntlich nur ein Statement folgen darf, denn sonst müßte man ein Ende der »WHILE«-Schleife definieren. »PROGRAM«&hellip; sind die Programmköpfe, »VAR«&hellip; die Deklarationsteile, auf die ich bald eingehen werde und innerhalb »BEGIN« und »END.« befindet sich das eigentliche Programm. »WRITE« beziehungsweise »WRITELN« entsprechen im Commodore-Basic »PRINT;« beziehungsweise »PRINT«, »READLN« dem »INPUT«. »SORT« entspricht »SQR« und berechnet die Quadratwurzel.</p>
```
[the remark this answers is not in the document]
Comment (mist64) [reply issues/8407/40 Pascal – leistungsfähiger und eleganter als Basic (Teil 1).html:77]: Yes.

**Comment by mist64** [comment]
Great, thank you so much.

> Also, there's are duplicate entries for "Basic Bär" (page 65), but I'm not sure which one you'd like to keep.

I don't understand...

**Comment by thierer** [comment]
> > Also, there's are duplicate entries for "Basic Bär" (page 65), but I'm not sure which one you'd like to keep.
> 
> I don't understand...

There's both [65 BASIC BÄR (The last one) – Ein Programmgenerator.html](https://github.com/mist64/64er-magazin.de/blob/issue8407/issues/8407/65%20BASIC%20B%C3%84R%20(The%20last%20one)%20%E2%80%93%20Ein%20Programmgenerator.html) and [65 Basic Bär (The last one) – Ein Programmgenerator.html](https://github.com/mist64/64er-magazin.de/blob/issue8407/issues/8407/65%20Basic%20B%C3%A4r%20(The%20last%20one)%20%E2%80%93%20Ein%20Programmgenerator.html).

File: issues/8407/40 Pascal – leistungsfähiger und eleganter als Basic (Teil 1).html (hunk 1 above)
[the remark this answers is not in the document]
Comment (mist64) [reply issues/8407/40 Pascal – leistungsfähiger und eleganter als Basic (Teil 1).html:77]: Ah, the README actually explains:

> Alle Artikel sind mit dem Text im gedruckten Magazin identisch, Schreibfehler und sachliche Fehler sind also unverändert. Errata aus späteren Ausgaben ("Fehlerteufelchen") werden den Artikeln allerdings angehängt, und später dokumentierte Fehler in Software sind in den Downloads bereits behoben.

**Pull request merged**